docs/data-cookbooks/cnm-workflow.md

```diff
@@ -0,0 +1,166 @@
+      Type: Task
+      Resource: ${SfSnsReportLambdaFunction.Arn}
+      CumulusConfig:
+        cumulus_message:
+          input: '{$}'
+      Next: HelloWorld
+    HelloWorld:
+      CumulusConfig:
+        buckets: '{$.meta.buckets}'
+        provider: '{$.meta.provider}'
+        collection: '{$.meta.collection}'
+      Type: Task
+      Resource: ${HelloWorldLambdaFunction.Arn}
+      Next: StopStatus
+    StopStatus:
+      Type: Task
+      Resource: ${SfSnsReportLambdaFunction.Arn}
+      CumulusConfig:
+        sfnEnd: true
+        stack: '{$.meta.stack}'
+        bucket: '{$.meta.buckets.internal.name}'
+        stateMachine: '{$.cumulus_meta.state_machine}'
+        executionName: '{$.cumulus_meta.execution_name}'
+        cumulus_message:
+          input: '{$}'
+      Catch:
+        - ErrorEquals:
+          - States.ALL
+          Next: WorkflowFailed
+      End: true
+    WorkflowFailed:
+      Type: Fail
+      Cause: 'Workflow failed'
+```
+If you're using the example deployment this workflow will already be included.  If you're attempting to use this example in a custom deployment you'll need to include the HelloWorld in your workflows.yml, then re-deploy to utilize it.
+
+### Task Configuration
+
+This cookbook entry assumes the HelloWorld [task](../workflows/developing-workflow-tasks.md) is defined in the `lambdas.yml` configuration file under `HelloWorld:`
+
+```
+HelloWorld:
+  handler: index.handler
+  timeout: 300
+  memory: 256
+  source: 'node_modules/@cumulus/hello-world/dist/'
+  useMessageAdapter: true
+```
+
+If you are using the example deployment this task will already be included in the configuration file.
+
+If you are attempting to use this example in a custom deployment you'll need to include the HelloWorld task in your lambdas.yml, then re-deploy to utilize it.
+
+##### Rule Configuration
+
+Cumulus provides a built-in Kinesis consumer lambda function ([kinesis-consumer](https://github.com/nasa/cumulus/blob/master/packages/api/lambdas/kinesis-consumer.js)) that will read events off of a preconfigured Kinesis stream and trigger a workflow when a Cumulus rule is configured via the Cumulus dashboard or API.
+
+This example will focus on using the Cumulus dashboard to schedule the execution of a HelloWorld workflow when events are posted to the Kinesis stream.
+
+To add a rule via the dashboard, navigate to the `Rules` page and click `Add a rule`, then configure the new rule using the following template (substituting correct values for parameters denoted by `${}`:
+
+```
+name: helloworld_cnm_rule
+Workflow Name: HelloWorldWorkflow
+Provider ID: ${provider_id} # found on the Providers page
+collection - Collection Name: ${collection_name} # configured and found in the Collections page
+collection - Collection Version: ${collection_version} # configured and found in the Collections page
+rule - type: kinesis
+rule - value: ${Kinesis_Stream_ARN} # See below
+Rule State: ENABLED
+Optional tags for search:
```

Comment:
It might be helpful here to include what the entry would look like in JSON in case someone wants to use the API... or maybe the API documentation will suffice here.